india_compliance/gst_india/utils/transaction_data.py

```diff
@@ -28,10 +28,17 @@ def __init__(self, doc):
         self.settings = frappe.get_cached_doc("GST Settings")
         self.transaction_details = frappe._dict()
 
+        gst_type = "Output"
+        if (
+            self.doc.doctype == "Purchase Invoice"
+            and self.doc.gst_category != "Unregistered"
```

Comment:
Instead, check for reverse charge.
No tax calculation is required for reverse charges.